apps/web/prisma/schema.prisma

```diff
@@ -256,3 +256,17 @@ model CouponCodes {
 
   @@index([redeemedById])
 }
+
+model ApiKey {
+  id        String   @id @default(cuid())
+  name      String
+  hashedKey String   @unique
+  validDays Int
```

Comment:
why in days? why not just an "end-date" validUntil or similar, so you wont calculate over and over again? even the database could easily filter for us..